.github/workflows/security-scan.yml

```
@@ -0,0 +1,87 @@
+name: dynamic-security-scan
+
+on:
+  schedule:
+    - cron: "45 8 * * 1" 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)